rest_framework_reactive/consumers.py

```diff
@@ -0,0 +1,29 @@
+from channels.generic.websockets import JsonWebsocketConsumer
+
+from .connection import get_subscriber_group_id
+from .client import QueryObserverClient
+
+
+class ObserversConsumer(JsonWebsocketConsumer):
```

Comment:
Does `JsonWebsocketConsumer` handles authentication?

Reply:
It can handle it, yes. But in the current example we don't actually need it as all user interaction comes through the standard REST API, where authentication is already handled. Authentication would need to be handled here if we supported incoming WebSocket messages.